packages/testing/src/common/types.ts

```diff
@@ -29,6 +29,13 @@ export type ClerkSetupOptions = {
    */
   secretKey?: string;
 
+  /*
+   * The backend API URL
+   * If not provided, the library will look for the key in the following environment variables:
+   * - CLERK_API_URL
+   */
+  apiUrl?: string;
+
```

Comment:
Can we not expose this publicly somehow? Because all users of the package should use the `api.clerk.com` 🤷 

Or, at least, mark it as `__internal_XXX`.